source/manual/running-rake-tasks.html.md

```diff
@@ -16,5 +16,3 @@ For example:
 ```sh
 kubectl exec deploy/publishing-api -- rake 'represent_downstream:published_between[2018-12-17T01:02:30, 2018-12-18T10:20:30]'
 ```
-
-The output of the command will be streamed to your terminal.
```

Comment:
I'm ignoring this accidental deletion. There is a story in an epic to review this documentation